javascript/javascript1/week2/homework.js

```diff
@@ -0,0 +1,106 @@
+// Flight booking fullname function----------------
+
+function getFullname(firstname, surname, useFormalName) {
+  if (firstname === "") {
```

Comment:
code if fine, alternatively read about 

- 'ternary condition' if you can reduce a few numbers of lines. 
- 'template string'